src/client/hud.cpp

```diff
@@ -504,6 +515,24 @@ void Hud::drawCrosshair()
 	}
 }
 
+void Hud::drawSelectionIndicator()
+{
+	int y_offset = 15;
+
+	if (use_selectionindicator_image) {
+		video::ITexture *selectionindicator = tsrc->getTexture("selectionindicator.png");
+		v2u32 size  = selectionindicator->getOriginalSize();
+		v2s32 lsize = v2s32(m_displaycenter.X - (size.X / 2),
+				m_displaycenter.Y - (size.Y / 2));
+		driver->draw2DImage(selectionindicator, lsize,
+				core::rect<s32>(0, 0, size.X, size.Y),
+				0, selectionindicator_argb, true);
```

Comment:
```suggestion
				nullptr, selectionindicator_argb, true);
```

Reply:
That part is code I copied. Changed both the 'copied' and 'copied from' code